server/Html.tsx

```diff
@@ -42,6 +44,12 @@ const Html = (props: Props) => {
 		return `${manifest[`${chunkName}.${extension}`]}`;
 	};
 
+	const activeTargetSlug =
+		props.initialData.scopeData.elements?.activeTarget.slug ||
+		(props.initialData.locationData.path.split('/')[1]
+			? props.initialData.locationData.path.substring(1).replace(/\//gi, '-')
```

Comment:
Can you provide an example of where this replacement might take effect? I see that it would turn `/foo/bar` into `foo-bar`, but I don't know where we'd see that happen in our URL scheme, and if it did I worry that it would quietly collide with something that actually has the slug `foo-bar`.